### PR TITLE
debian: Add player into syncd package

### PR DESCRIPTION
**Pull request by stcheng**

### CODE REVIEW

File: debian/syncd.install [hunk 1]
```diff
@@ -1,2 +1,3 @@
 usr/bin/syncd*
+usr/bin/player*
```
Comment (qiluo-msft) [inline debian/syncd.install]: Just one concern, is this name too general?

Comment (stcheng) [inline debian/syncd.install]: what about record-player?

Comment (lguohan) [inline debian/syncd.install]: sai-player?

**Comment by stcheng** [comment]
updated.

File: saiplayer/Makefile.am [hunk 2]
```diff
@@ -0,0 +1,13 @@
+AM_CPPFLAGS = -I/usr/include/sai
```
Comment (qiluo-msft) [inline saiplayer/Makefile.am]: You moved the file, and delete one -I, which break the build

**Pull request merged**